crates/oxc_linter/src/ast_util.rs

```diff
@@ -201,7 +201,7 @@ pub fn get_enclosing_function<'a, 'b>(
         {
```

Comment:
It isn't a change request, Mostly thinking out loud; While you are at it maybe we can also replace this condition with `current_node.kind().is_function_like()` for the sake of consistency.

Reply:
This sounds like a good idea; I can make a separate PR.